src/app/store/slices/likedMoviesSlice.js

```diff
@@ -0,0 +1,22 @@
+import { createSlice } from '@reduxjs/toolkit';
+
+const likedMoviesSlice = createSlice({
+	name: 'likedMovies',
+	initialState: [],
+	reducers: {
+		addMovie: (state, action) => {
```

Comment:
`addMovie` both adds and removes movies, this makes the intent of `addMovie` unclear
rename to something like `toggleMovie`